general/forall.hpp

```diff
@@ -116,20 +116,20 @@ void OmpWrap(const int N, HBODY &&h_body)
 /// RAJA Cuda and Hip backends
 #if defined(MFEM_USE_RAJA) && defined(RAJA_ENABLE_CUDA)
 using cuda_launch_policy =
-   RAJA::expt::LaunchPolicy<RAJA::expt::null_launch_t, RAJA::expt::cuda_launch_t<true>>;
+   RAJA::LaunchPolicy<RAJA::cuda_launch_t<true>>;
```

Comment:
Do we want to maintain the support for older versions of RAJA?

Reply:
ping: @artv3, @kennyweiss ☝️

Reply:
I chatted with @kennyweiss, and since we are moving RAJA::launch out of expt it may make sense to encourage people to move to the more stable API now. In addition its easier to maintain one API.

Reply:
thanks for clarifying this